api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -15,18 +15,34 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
+import org.junit.Before;
```

Comment:
please organize the imports with your IDE. choose your IDE here https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE and look for "Java Imports"